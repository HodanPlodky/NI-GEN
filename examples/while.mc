int fib(int n) {
    if (n <= 0)
        return 0;
    else if (n <= 1)
        return 1;
    else
        return fib(n - 1) + fib(n - 2);
}

int betterfib(int n) {
    int a = 0;
    int b = 1;
    return a;
}

int main() {
    int i = 0;
    int sum = 0;
    while (i < 10) {
        sum = sum + fib(i + 10);
        i = i + 1;
    }
    return sum;
}
