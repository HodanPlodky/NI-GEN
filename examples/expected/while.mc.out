input file : examples/tests/while.mc
return 105
